setup.cfg

```diff
@@ -8,5 +8,3 @@ requires-dist =
     pyOpenSSL >= 0.13; python_version < '3.0'
     requests >= 2.1.0
     service_identity >=14.0.0
-    Twisted >= 15.5.0; python_version > '3.0'
-    Twisted >= 14.0.2; python_version < '3.0'
```

Comment:
This now conflicts with the cleanup I did in #155, but that should be simple to resolve.